miden-crypto/src/ies/keys.rs

```diff
@@ -283,6 +337,45 @@ impl UnsealingKey {
     }
 }
 
+impl Serializable for UnsealingKey {
```

Comment:
Nit: do we need `Serializable` as an accessible code path, or could we get away with making this `#[cfg(test)]`? I like the idea of private keys not able to go somewhere they shouldn't. Just a thought.

Reply:
These are valid points but I feel that they touch almost all of the crate in some way or another, should we open an issue to discuss this?
I will leave this as is for the time being until we figure out a global solution